llvm/unittests/CodeGen/SelectionDAGPatternMatchTest.cpp

```diff
@@ -125,6 +125,7 @@ TEST_F(SelectionDAGPatternMatchTest, matchVecShuffle) {
   auto VInt32VT = EVT::getVectorVT(Context, Int32VT, 4);
   const std::array<int, 4> MaskData = {2, 0, 3, 1};
   const std::array<int, 4> OtherMaskData = {1, 2, 3, 4};
+  ArrayRef<int> Mask(MaskData);
```

Comment:
Once `m_SpecficMask` takes ArrayRef rather than `ArrayRef &`, you don't need the explicit declaration of `Mask` here since `MaskData` will be implicitly converted to ArrayRef below.